src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: What is web3 and why is it important?
+description: An introduction to web3 - the next evolution of the internet - and why it matters.
+lang: en
+sidebar: true
+---
+
+# Introduction to web3 {#introduction}
+
+Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, undemocratically deciding what should and should not be allowed.
```

Comment:
I'm not a fan of using the term "undemocratically" here as it implies that web3 provides a democratic solution to the problem (which it definitely doesn't).  Consider instead:
```suggestion
Centralization has helped onboard billions of people to the internet and created the stable, robust infrastructure on which it lives. At the same time, a handful of centralized entities have a stronghold on large swathes of the internet, unilaterally deciding what should and should not be allowed.
```